src/session.ts

```diff
@@ -76,10 +77,6 @@ export const PowerShellVersionRequestType =
     new RequestType0<IPowerShellVersionDetails, void>(
         "powerShell/getVersion");
 
-export const RunspaceChangedEventType =
-    new NotificationType<IRunspaceDetails>(
-        "powerShell/runspaceChanged");
```

Comment:
@SeeminglyScience this notification isn't wired up, and I wasn't planning to rewire it, so I deleted it. What do you think?

Reply:
If it's not used I'm with ya for sure. It's in git history if we need it again 👍